src/core/ext/filters/client_channel/lb_policy.h

```diff
@@ -62,6 +62,16 @@ class LoadBalancingPolicy
 
   /// State used for an LB pick.
   struct PickState {
+    PickState()
```

Comment:
In the normal pick path, these initializations are not needed, because the entire call stack is getting memset to 0, and we're not going through C++ initialization at all, since these are being treated like C structs.  That's why I handled this differently in #16207: I didn't want to take on the additional overhead of setting these fields to 0 twice in the call path.

Let's get my PR merged first, then we can do the rest of this.